_data/sites.json

```
@@ -1701,6 +1701,21 @@
         ]
     },
 
+    {
+        "name": "Auchan",
+        "url": "https://www.auchan.fr/faq/section/mon-compte-client/mon-compte-auchan-fr?parentNodeId=adf9b2fc-cd05-49df-9833-70766279aea3#anchor_3bea6236-a7b1-4c98-a754-fb04d325831c",
+        "difficulty": "hard",
+        "notes": "Send an email to the address given here. It's not listed on the FAQ page.",
+        "notes_fr": "Envoyez un courriel à l'adresse e-mail donné sur ce site. L'adresse n'est pas donné sur le site d'Auchan.",
+        "domains": [
+            "auchan.fr",
+            "www.auchan.fr"
```

```suggestion
            "auchan.fr"
```
The domain with 'www' is not needed